### PR TITLE
Update apache2.md

### PR DESCRIPTION
**Pull request by 343334**
update conf and domains to rocketmap to keep inline with the rest of the application.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/extras/apache2.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Assuming the following:
 
    </VirtualHost>
    ```
-   If you want your maps at `pokemongo.yourdomain.com`, keep it just like it is
+   If you want your maps at `rocketmap.yourdomain.com`, keep it just like it is
    If you want your maps at `yourdomain.com/go/` (note the trailing slash!)
```
Comment (myntath) [inline docs/extras/apache2.md]: Perhaps we should change /go/ as well. I would suggest /rm/ or /RocketMap/

Comment (Arengo) [inline docs/extras/apache2.md]: ``/map/`` ?

Comment (myntath) [inline docs/extras/apache2.md]: even better, and has me laughing because that is exactly where I run my own and I didn't think of it.

**Comment by FrostTheFox** [comment]
ah my bad technically should've merged this over #1954 ¯\\\_(ツ)_/¯

**Pull request closed without merging**